File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux, luy, rdx, rdy = 100, 100, 0, 0
    for n_row, row in enumerate(wallpaper):
        for n_col, r in enumerate(row):
            if r == '#':
                lux = min(lux, n_row)
                rdx = max(rdx, n_row+1)
                luy = min(luy, n_col)
                rdy = max(rdy, n_col+1)
    
    answer = [lux, luy, rdx, rdy]
    
    return answer

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.swift
import Foundation

/**
빈칸: ., 파일: #
*/
func solution(_ wallpaper:[String]) -> [Int] {
    let wallpaper = wallpaper.map { Array($0) }
    var (lux, luy) = (Int.max, Int.max)     // 최소값 찾아야 됨.
    var (rdx, rdy) = (Int.min, Int.min)     // 최댓값 찾아야 함.
    
    for row in 0..<wallpaper.count {
        for col in 0..<wallpaper[0].count {
            if wallpaper[row][col] == "#" {
                lux = min(lux, row)
                luy = min(luy, col)
                rdx = max(rdx, row + 1)
                rdy = max(rdy, col + 1)
            }
        }
    }

    return [lux, luy, rdx, rdy]
}